values.yaml

```diff
@@ -547,6 +650,15 @@ connectInject:
   # method for Connect inject, set this to the name of your auth method.
   overrideAuthMethodName: ""
 
+  # aclInjectToken refers to a Kubernetes secret that you have created that contains
+  # an ACL token for your Consul cluster which allows the Connect injector the correct
+  # permissions. This is only needed if Consul namespaces and ACLs are enabled on the
+  # Consul cluster and you are not setting `global.bootstrapACLs` to `true`.
+  # This token needs to have `operator = "write"` privileges.
```

Comment:
It'd be nice to explain why we need elevated `operator:write` perms here. Also, this is only needed if namespaces are enabled + plus the other conditions you've mentioned, right?

Reply:
From the comment, "This is only needed if Consul namespaces and ACLs are enabled on the
 Consul cluster and you are not setting `global.bootstrapACLs` to `true`"

Added comment about the token privileges.